data/kitchen/prkitchen_dishwasher.kinbody.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" ?>
+<KinBody name="prkitchen_dishwasher">
+    <Body name="dish_base" type="dynamic">
+        <Mass>
+            <total>84.485646567076898</total>
+            <inertia>11.928260 -0.000000 -0.201267 -0.000000 10.880212 0.000000 -0.201267 0.000000 6.380171</inertia>
+            <com>-0.017193 -0.000000 0.338541</com>
+        </Mass>
+        <Geom render="false" type="trimesh">
+            <Data>/homes/jeking/ros/local/collision_checking/pr_kitchen/meshes/dish_base.wrl</Data>
```

Comment:
There are a bunch of absolute mesh paths in here. Could you replace them with relative paths?